hasura/config.yaml

```diff
@@ -1,4 +1,4 @@
 version: 2
 endpoint: http://localhost:15003
-admin_secret: 2OWslm5aAjlTARU
+admin_secret: ${admin_secret}
```

Comment:
Hi @sourabhsahu1, is this intentional ?

I dont think there would be anything apart from build folder that we need.

Reply:
Hi @tushar5526  Last time I forked this repo and forgot to remove admin_secret while raising PR (radhay told me to remove it , therefore i remember)  and this time admin_secret is visible  on master branch so removed it .

Reply:
Cool, just as a side note this secret would still be accessible and you might have to force push or remove it from the history. 

Merging for now.